test/rules/give_deprecation_message.dart

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year

Reply:
FWIW: I'm OK with whatever you decide about the date; I might lean towards 2019 but don't care strongly either way.

Reply:
Done. 2019.